src/wizard/javascript/remix/with-error-monitoring-performance-and-replay.md

```diff
@@ -0,0 +1,118 @@
+---
+name: Remix
+doc_link: https://docs.sentry.io/platforms/javascript/guides/remix/
+support_level: production
+type: framework
+---
+
+## Install
+Sentry captures data by using an SDK within your application’s runtime.
+
+```bash
+# Using yarn
+yarn add @sentry/remix
+
+# Using npm
+npm install --save @sentry/remix
+```
+
+## Configure
+Import and initialize Sentry in your Remix entry points for both the client and server:
+
+```javascript
+import { useLocation, useMatches } from "@remix-run/react";
+import * as Sentry from "@sentry/remix";
+import { useEffect } from "react";
+
+Sentry.init({
+  dsn: "___DSN___",
+  integrations: [
+    new Sentry.BrowserTracing({
+      routingInstrumentation: Sentry.remixRouterInstrumentation(
+        useEffect,
+        useLocation,
+        useMatches
+      ),
+    }),
+    new Sentry.Replay(),
+  ],
+  // Performance Monitoring
+  tracesSampleRate: 1.0,
+  // Session Replay
+  replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
+  replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
+});
+```
+
+Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/v1/api/conventions#action) and [`loader`](https://remix.run/docs/en/v1/api/conventions#loader) functions. You can also initialize Sentry's database integrations, such as Prisma, to get spans for your database calls:
+
+```javascript
+import { prisma } from "~/db.server";
+
+import * as Sentry from "@sentry/remix";
+
+Sentry.init({
+  dsn: "___DSN___",
+  integrations: [
+    new Sentry.Integrations.Prisma({ client: prisma }),
```

Comment:
Is one `Sentry.Integrations.` and the other one (below) is not?

Reply:
Yes this is correct.